_rules/no-auto-play-audio-80f0bf.md

```diff
@@ -1,9 +1,9 @@
 ---
 id: 80f0bf
-name: Video or audio has no auto-play audio
+name: 'Video or audio has no `autoplay` audio'
```

Comment:
If autoplay is used as an adjective (?) or a verb, there is no reason for the backticks. In these cases, autoplay does not refer to the autoplay attribute.

To avoid confusion, consider rewriting where autoplay does not refer to the attribute.

This also goes for the other uses of the word autoplay in this rule.